include/jsonbuilder/JsonBuilder.h

```diff
@@ -1572,11 +1674,10 @@ JSON_DECLARE_JsonImplementType(signed long long, );
 JSON_DECLARE_JsonImplementType(float, );
 JSON_DECLARE_JsonImplementType(double, );
 
+JSON_DECLARE_JsonImplementType(TimeStruct, );
 JSON_DECLARE_JsonImplementType(std::chrono::system_clock::time_point, );
-JSON_DECLARE_JsonImplementType(UuidStruct, );
+JSON_DECLARE_JsonImplementType(UuidStruct, const&);
 JSON_DECLARE_JsonImplementType(std::string_view, );
-JSON_DECLARE_JsonImplementType_AddValue(std::string);
-JSON_DECLARE_JsonImplementType_AddValue(char);
```

Comment:
Note: Minor breaking changes.
- remove support for `builder.push_back(std::string)`. Support requires us to `#include <string>`, which some customers might not want to do. And it's really easy to work around this -- just do `builder.push_back<std::string_view>(string)` instead.
- remove support for `builder.push_back(char)`. The `char` type (text) is very easily mixed-up with `unsigned char` (UINT8) and `signed char` (INT8). In addition, it is easy to forget that this will only support char values 0..127 (there is no such thing as a single UTF-8 char value 128..255). Instead, if we really need a char, use a string_view of length 1.